src/components/CheckEditor/CheckProbes/PrivateProbesAlert.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { Alert, LinkButton, Stack } from '@grafana/ui';
+
+import { ROUTES } from 'types';
+import { getRoute } from 'components/Routing.utils';
+
+export const PrivateProbesAlert = () => {
+  const [display, setDisplay] = useState(true);
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <Alert
+      title="You haven't set up any private probes yet."
+      severity="info"
```

Comment:
The original design used a different color and icon for this message, but with the intention of reusing and not modifying the elements we get from `grafana/ui` for consistency with other parts of the app, I've set a `info` severity which displays in light blue. Happy to adapt it to the original design if you think that's better though.

![image](https://github.com/user-attachments/assets/e208a8bf-0acb-4bef-8612-f5169e479438)
![image](https://github.com/user-attachments/assets/5468b0f8-d31b-445c-b43c-a3a365eb9e08)